Assets/HoloToolkit/UX/Scripts/Progress/ProgressIndicator.cs

```diff
@@ -29,8 +29,8 @@ public bool IsLoading
         private IndicatorStyleEnum defaultIndicatorStyle = IndicatorStyleEnum.AnimatedOrbs;
         [SerializeField]
         private ProgressStyleEnum defaultProgressStyle = ProgressStyleEnum.Percentage;
-        [SerializeField]
-        private ProgressMessageStyleEnum defaultMessageStyle = ProgressMessageStyleEnum.Visible;
+        //[SerializeField]
+        //private ProgressMessageStyleEnum defaultMessageStyle = ProgressMessageStyleEnum.Visible;
```

Comment:
@cre8ivepark or @johnppella, is this supposed to be used? Looks like it's not. Are there planned but unimplemented features here?

Reply:
oneButtonSet is required when the dialog is going to have only one button.
The oneButtonSet and twoButtonSet variables do not need to be flagged with SerializeField
since they are currently not set in the editor in the example ( this is a leftover from the original implementation from which this example was ported).. They can also be defined in the GenerateButtons() function since they are only used within that func.